code/game/machinery/doors/firedoor.dm

```diff
@@ -217,27 +221,6 @@
 				continue
 			process_results(checked_turf)
 
-/obj/machinery/door/firedoor/proc/register_adjacent_turfs(atom/loc)
-	for(var/dir in GLOB.cardinals)
-		var/turf/checked_turf = get_step(get_turf(loc), dir)
-
-		if(!checked_turf)
-			continue
-		process_results(checked_turf)
-		RegisterSignal(checked_turf, COMSIG_TURF_EXPOSE, .proc/process_results)
-		RegisterSignal(checked_turf, COMSIG_TURF_CALCULATED_ADJACENT_ATMOS, .proc/process_results)
-
```

Comment:
Nah go back uh, just move the register turfing for `COMSIG_TURF_CALCULATED_ADJACENT_ATMOS` from the adjacent turf to just get_turf(loc)

We still need the looping for exposure

Reply:
This work?